requirements-test.txt

```diff
@@ -10,3 +10,4 @@ sphinx
 sphinx_rtd_theme
 freezegun
 openwisp-utils>=0.2,<0.3
+django-crispy-forms
```

Comment:
bind its requirements as with `openwisp-utils`

Reply:
I sent you a few messages in the public chat regarding this, if not required I suggest to remove it because doesn't seeem to bring much value and will slow down the build by increasing the build time